drivers/gles3/shaders/scene.glsl

```diff
@@ -1466,17 +1466,32 @@ void main() {
 		const float c3 = 0.743125;
 		const float c4 = 0.886227;
 		const float c5 = 0.247708;
-		ambient_light += (c1 * lightmap_captures[8].rgb * (wnormal.x * wnormal.x - wnormal.y * wnormal.y) +
-								 c3 * lightmap_captures[6].rgb * wnormal.z * wnormal.z +
-								 c4 * lightmap_captures[0].rgb -
-								 c5 * lightmap_captures[6].rgb +
-								 2.0 * c1 * lightmap_captures[4].rgb * wnormal.x * wnormal.y +
-								 2.0 * c1 * lightmap_captures[7].rgb * wnormal.x * wnormal.z +
-								 2.0 * c1 * lightmap_captures[5].rgb * wnormal.y * wnormal.z +
-								 2.0 * c2 * lightmap_captures[3].rgb * wnormal.x +
-								 2.0 * c2 * lightmap_captures[1].rgb * wnormal.y +
-								 2.0 * c2 * lightmap_captures[2].rgb * wnormal.z) *
+		vec3 c = (c1 * lightmap_captures[8].rgb * (wnormal.x * wnormal.x - wnormal.y * wnormal.y) +
+						 c3 * lightmap_captures[6].rgb * wnormal.z * wnormal.z +
+						 c4 * lightmap_captures[0].rgb -
+						 c5 * lightmap_captures[6].rgb +
+						 2.0 * c1 * lightmap_captures[4].rgb * wnormal.x * wnormal.y +
+						 2.0 * c1 * lightmap_captures[7].rgb * wnormal.x * wnormal.z +
+						 2.0 * c1 * lightmap_captures[5].rgb * wnormal.y * wnormal.z +
+						 2.0 * c2 * lightmap_captures[3].rgb * wnormal.x +
+						 2.0 * c2 * lightmap_captures[1].rgb * wnormal.y +
+						 2.0 * c2 * lightmap_captures[2].rgb * wnormal.z) *
 				scene_data.emissive_exposure_normalization;
+
+#if defined(SPECULAR_OCCLUSION) || defined(SPECULAR_OCCLUSION_CONSERVATIVE)
+		float specular_occlusion = (c.r * 0.3 + c.g * 0.59 + c.b * 0.11) * 2.0; // brightness from lightmap color
```

Comment:
```suggestion
		float specular_occlusion = (c.r * 0.3 + c.g * 0.59 + c.b * 0.11) * 2.0; // Brightness from lightmap color.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).